### ISSUE TITLE
[Bug]: Unable to grep the tests with tag annotation

### ISSUE THREAD

**Issue by Vaideeshwaran-Saravanabhavan-ZS0572** [microsoft/playwright#29955]
### Version

1.41.2

### Steps to reproduce

test("testing the product detail page", {
    **tag: '@feature',**
}, async({page})=> { 
//steps go here

})

### Expected behavior

I expect to run the test with the annotated tag @feature like below:
npx playwright test -g '@feature'

### Actual behavior

But the playwright says no test found.

```
PS > npx playwright test -g '@feature'
Error: No tests found
```

### Environment

```shell
System:
OS : Windows 11 pro 64-bit operating system, x64-based processor

Binaries:
Node : v16.17.1 

npmPackage:
"@playwright/test": "^1.41.2",
```

**Comment by mxschmitt** [comment]
The new native tags were released in v1.42 - have you tried upgrading to v1.42?

**Comment by Vaideeshwaran-Saravanabhavan-ZS0572** [comment]
Thanks It is working after upgrade to 1.42
